benchmark_runner/workloads/winmssql_vm.py

```
@@ -62,22 +35,106 @@ def run(self):
+                check = self._virtctl.virtctl_ssh(vm_name=self._vm_name, command=check_cmd,
+                                                  namespace=self.__namespace, key_path=self.__ssh_key_path, username=self.__username)
+                if check and 'found' in check:
+                    break
+                logger.info(f'Waiting for hammerdb_result.json on VM... ({elapsed}s)')
+                time.sleep(OC.DELAY)
+            if self._virtctl._scp_file(vm_name=self._vm_name, remote_path=f'{self.__remote_dir}/results/hammerdb_result.json',
+                                       local_path=local_result_json,
+                                       namespace=self.__namespace, key_path=self.__ssh_key_path, username=self.__username):
+                logger.info(f'hammerdb_result.json copied to {local_result_json}')
+            else:
+                logger.warning(f'Failed to SCP hammerdb_result.json from VM')
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)